Examples/Extensions/TodayWidget/Base.lproj/MainInterface.storyboard

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This storyboard was generated as part of the addition of an example widget target to see MapboxMobileEvents running in an extension.